test/runtests.jl

```diff
@@ -75,7 +75,6 @@ CUDA.allowscalar() do
         @testset "Poisson Solvers 2" begin
             include("test_poisson_solvers_stretched_grids.jl")
             include("test_conjugate_gradient_poisson_solver.jl")
-            include("test_poisson_solvers_immersed_boundaries.jl")
```

Comment:
These were moved to `test_conjugate_poisson_solver.jl`, so that all tests for the CGSolver are in one place